packages/expo-notifications/android/src/main/java/expo/modules/notifications/scoper/BareStringScoper.java

```diff
@@ -0,0 +1,28 @@
+package expo.modules.notifications.scoper;
+
+import org.unimodules.core.interfaces.InternalModule;
+
+import java.util.Collections;
+import java.util.List;
+
+public class BareStringScoper implements InternalModule, StringScoper {
```

Comment:
Let's not add a notion of scoping to bare apps. I think we can safely assume that even in brownfield apps, where one may have more than one React Native application we can shift the responsibility of scoping strings on the developer. `BareStringScoper` and any scoping that is happening in `NotificationsModule` should happen in `android/expoview`, where we will override `NotificationsModule` with `ScopingNotificationsModule < NotificationsModule`.

Reply:
I think that ScoperModule helps here. If we want to add scoping all we need to do is override this internal module.

Reply:
True, but we don't want to oblige authors of native modules to use any scoping of names — they should be able to implement features in a non-scoping way. Scoping should be the responsibility of the person that adds the need to scope, so either a developer that runs more RN instances side-by-side (one would need to ensure that names in these RN apps aren't the same) or of Expo which can override necessary modules in expoview.